cmake/configure_apps.cmake

```diff
@@ -13,6 +13,8 @@
 ###############################################################################
 ### Configure Application Components
 ###############################################################################
+string(TOUPPER ${APP} APP)
```

Comment:
Again same with top level CMake file. Let's keep changes minimal in these files. If we need to clean them and make it more flexible and readable. The first thing is to push those changes to ufs-weather-model. We will discuss it more in the meeting with EMC folks next week.